### PR TITLE
Integrate gutenberg-mobile release 1.47.0

### PR DESCRIPTION
**Pull request by antonis**
## Description
This PR incorporates the 1.47.0 release of gutenberg-mobile.  
For more information about this release and testing instructions, please see the related Gutenberg-Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/3155

Release Submission Checklist

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Review by dcalhoun — COMMENTED** [review]
This is looking good to me so far. We'll need to wait until we can fix the build issue before starting manual testing.

**Comment by antonis** [comment]
At this stage this LGTM 🎉

**Review by dcalhoun — APPROVED** [review]
LGTM. Before we merge, we just need to update the `libs/gutenberg` submodule to reference the merge commit after we merge wordpress-mobile/gutenberg-mobile#3155.

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 16.8
 -----
+* [*] Block Editor: Fixed block mover title wording for better clarity from 'Move block position' to 'Change block position'. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3049]
+* [**] Block Editor: Make inserter long-press options "add to beginning" and "add to end" always available. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3074]
```
Comment (fluiddot) [inline RELEASE-NOTES.txt:6]: This change was included in the 1.46.1 beta fix so it should better go under 16.7 section.

File: RELEASE-NOTES.txt [hunk 2]
```diff
@@ -2,6 +2,10 @@
...
+* [**] Block Editor: Add support for setting Cover block focal point. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3028]
```
Comment (fluiddot) [inline RELEASE-NOTES.txt:7]: This change was included in the 1.46.1 beta fix so it should better go under 16.7 section.

File: RELEASE-NOTES.txt [hunk 3]
```diff
@@ -2,6 +2,10 @@
...
+* [*] Block Editor: Fix crash when Column block width attribute was empty. [https://github.com/WordPress/gutenberg/pull/29015]
```
Comment (fluiddot) [inline RELEASE-NOTES.txt]: Same as previous comment, this change was included in the 1.46.1 beta fix so it should better go under 16.7 section.

**Review by fluiddot — COMMENTED** [review]
👋 I've found some discrepancies in the release notes related to the changes included the beta fixes.

File: RELEASE-NOTES.txt (hunk 1 above)
Comment (fluiddot) [inline RELEASE-NOTES.txt]: This change was included in the 1.46.1 beta fix so it should better go under 16.7 section.

File: RELEASE-NOTES.txt (hunk 3 above)
Comment (antonis) [reply RELEASE-NOTES.txt]: Thank you for the heads up @fluiddot 🙇 
Issue fixed with https://github.com/wordpress-mobile/WordPress-Android/pull/14081/commits/e7298e35489509ce817bb859e9ba572689d0d806

File: RELEASE-NOTES.txt (hunk 1 above)
Comment (antonis) [reply RELEASE-NOTES.txt]: Issue fixed with https://github.com/wordpress-mobile/WordPress-Android/pull/14081/commits/e7298e35489509ce817bb859e9ba572689d0d806

**Review by fluiddot — APPROVED** [review]

**Pull request merged**